editions/tw5.com/tiddlers/Breadcrumbs Macros.tid

```diff
@@ -0,0 +1,96 @@
+breadcrumbs: aa bb cc dd ee
+created: 20230408131424728
+modified: 20230425174048480
+tags: 
+title: Breadcrumbs Macros
+type: text/vnd.tiddlywiki
+
+!! Introduction
+
+[[Breadcrumbs]] are used as a navigational aid in user interfaces and on web pages. It allows users to keep track and maintain awareness of their locations within programs, documents, or websites.
+
+There are 4 documentation `.breadcrumbs`-related macros, which are designed for different usecaseses.
+
+|`.breadcrumbs` |A wrapper macro that calls "-Story" (default) "-Simple" or "-Field", depending on the <<.param mode>> parameter |
+|`.breadcrumbsStory` |By default, this recursive macro uses the ''tags'' field to create the breadcrumbs structure. Opening the links recreates this structure in the story-list as close as possible |
+|`.breadcrumbsSimple` |This recursive macro also uses the ''tags'' field to define the breadcrumbs structure. Opening linked tiddlers uses the wiki default settings  |
+|`.breadcrumbsField` |By default this macro uses the ''list'' field to create the structure. This macro is ''not recursive''. Opening the links recreates this structure in the story-list as close as possible  |
+
+!! Mode Parameter
+
+With the <<.param mode>> parameter, the `<<.breadcrumbs mode:"story">>` macro is used to switch between "story", "simple" and "field" mode. The `.breadcrumbs` macro is a "wrapper", to allow switching mode using global settings. It is the only macro with the <<.param mode>> parameter. 
+
+|<<.param mode>> |Defaults to "story". Allows <<.value story>>, <<.value simple>> or <<.value field>> |
+
+!!! Mode Values
+
+* <<.value story>> ... Clicking on a link will open the new tiddler according to the position within the breadcrumbs hierarchy. So the hierarchy is reflected by the position in the story-river
```

Comment:
This is not correct usage of an ellipsis – they are used to indicate that material has been omitted. It would be more consistent to use a colon: `<<.value story>>: clicking...`

Reply:
changed that